compiler/hash-typecheck/src/storage/core.rs

```diff
@@ -87,7 +87,7 @@ impl CoreDefs {
 
         // Any type
         let any_ty = builder.create_any_ty_term();
-        builder.add_pub_member_to_scope("Type", builder.create_trt_kind_term(), any_ty);
+        builder.add_pub_member_to_scope("AnyType", builder.create_trt_kind_term(), any_ty);
```

Comment:
Maybe just `Any`

Reply:
I dont think thats a good idea, since `Trait` for example is not assignable to `AnyType`, and `Any` sounds like it should be anything,